### PR TITLE
Add support for is array for rep commands js

### PR DESCRIPTION
**Pull request by YuvHayun**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CIAC-9047](https://jira-dc.paloaltonetworks.com/browse/CIAC-9047)

### CODE REVIEW

File: Packs/IsItPhishing/Integrations/IsItPhishing/IsItPhishing.js [hunk 1]
```diff
@@ -29,87 +29,92 @@ var sendRequest = function(method, api, body) {
...
-    var res = sendRequest('POST', 'check', body);
-    var ec = {
-        IsItPhishing: {Url: url},
-        DBotScore: {
-            Indicator: url,
-            Score: 0,
-            Type: 'url',
-            Vendor: 'IsItPhishing',
-            Reliability: params.integrationReliability
+    var urls = url.split(',')
```
Comment (JudahSchwartz) [inline Packs/IsItPhishing/Integrations/IsItPhishing/IsItPhishing.js:32]: we dont have any argtolist function? Is this how we're doing it in other js reputation commands?

File: Packs/IsItPhishing/ReleaseNotes/1_0_4.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### IsItPhishing
+
+- Added support for multiple url addresses input for **url** command.
```
Comment (JudahSchwartz) [inline Packs/IsItPhishing/ReleaseNotes/1_0_4.md:6]:
```suggestion
- Added support for multiple url address inputs for the **url** command.
```

File: Packs/Pipl/ReleaseNotes/1_0_9.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Pipl
+
+- Added support for multiple email addresses input for **email** command.
```
Comment (JudahSchwartz) [inline Packs/Pipl/ReleaseNotes/1_0_9.md:6]:
```suggestion
- Added support for multiple email address inputs in the **email** command.
```

File: Packs/isight/Integrations/FireEyeISIGHT/FireEyeISIGHT.js [hunk 4]
```diff
@@ -129,117 +129,144 @@ var createContextReportsAndScore = function(records) {
...
-                    Vendor: VENDOR_NAME,
-                    Score: 0,
-                    Reliability: params.integrationReliability
+    var ips = ip.split(',')
+    var results = new Array(urls.length)
+    for (var i = 0; i < ips.length; i++) {
+        var records = basicSearch('ip', ips[i]);
+
+        if (!records) {
+            return {
```
Comment (JudahSchwartz) [inline Packs/isight/Integrations/FireEyeISIGHT/FireEyeISIGHT.js:138]: returning here and not adding to array? Will only return first one

**Review by JudahSchwartz — COMMENTED** [review]
looks good, check the comment i added

File: Packs/IsItPhishing/Integrations/IsItPhishing/IsItPhishing.js (hunk 1 above)
Comment (YuvHayun) [reply Packs/IsItPhishing/Integrations/IsItPhishing/IsItPhishing.js:32]: Yap

**Review by JudahSchwartz — APPROVED** [review]

**Comment by YuvHayun** [comment]
@ShahafBenYakir need force merge on this ticket, getting errors for missing TPBs and skipped tests.

**Pull request merged**